### PR TITLE
Add tool to automatically create a new release

### PR DESCRIPTION
**Pull request by sbrunner**
See the README changes :-)

### CODE REVIEW

File: c2cciutils/scripts/version.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+            ],
+        )
+
+    if os.path.exists(".github/renovate.json5"):
+        with mra.EditRenovateConfig(".github/renovate.json5") as renovate_config:
+            if stabilization_branches:
+                if "baseBranches" in renovate_config.data:
+                    renovate_config.data = re.sub(
+                        r"^( +baseBranches: \[)(.*)\]",
+                        rf"\1{json.dumps(stabilization_branches_with_master)}",
```
Comment (sbrunner) [inline c2cciutils/scripts/version.py:131]: Should be convert in JSON

File: c2cciutils/scripts/version.py [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+import os
+import re
+import subprocess  # nosec
+
+import multi_repo_automation as mra
+import ruamel.yaml.comments
+
+
+def main() -> None:
+    """Create a new version with his stabilization branch."""
```
Comment (vuilleumierc) [inline c2cciutils/scripts/version.py:14]:
```suggestion
    """Create a new version with its stabilization branch."""
```

File: c2cciutils/scripts/version.py [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+
+    args_parser = argparse.ArgumentParser(
+        description="Create a new version with his stabilization branch",
```
Comment (vuilleumierc) [inline c2cciutils/scripts/version.py:17]:
```suggestion
        description="Create a new version with its stabilization branch",
```

**Review by vuilleumierc — APPROVED** [review]

**Comment by sbrunner** [comment]
Thanks @vuilleumierc

**Pull request merged**